pageCounter.py

```
@@ -1,220 +1,221 @@
+    
+    # Sort files by page count
+    sorted_files = sorted(files_with_pages, key=lambda x: x[1])
+    
+    # Display results
+    logging.info("\nFiles sorted by page count:")
+    total_pages = 0
+    for filename, pages in sorted_files:
+        logging.info(f"{filename}: {pages} pages")
+        total_pages += pages
+    
+    logging.info(f"\nSummary:")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)